doc/markdown/Intructions.md

```diff
@@ -21,6 +21,46 @@ Response: Where is it located?
 
 ## Dataset management functions
 
+### Subset of dataset
+
+#### \- By row
+
+* select by rows from **1** to **10** from **energy**
```

Comment:
I would reformulate this sentence to:

select rows from **energy** starting at **1** until **10**